### PR TITLE
added new ecs 1.9 fields to lowercase, related hosts and related ip enrichments

### PR DESCRIPTION
**Pull request by Akhila-Y**
## Description
added new ecs 1.9 fields to lowercase, related hosts and related ip enrichments

## Related Issues
NA

## Todos
NA

### CODE REVIEW

File: config/enrichments/07_lowercase.conf [hunk 1]
```diff
@@ -264,15 +277,21 @@ filter {
         "group.id",
         "group.name",
         "host.architecture",
+        "host.cpu.usage",
+        "host.disk.read.bytes",
```
Comment (KrishnanandSingh) [inline config/enrichments/07_lowercase.conf:281]: This is a long field. Lowercasing would work only for text/keyword fields.

File: config/enrichments/07_lowercase.conf [hunk 2]
```diff
@@ -47,13 +46,16 @@ filter {
         "client.bytes",
         "client.domain",
         "client.geo.city_name",
+        "client.geo.continent_code",
```
Comment (KrishnanandSingh) [inline config/enrichments/07_lowercase.conf:49]: These are standard uppercase. I'm not sure if we should lowercase these.
@brian-grabau  should we lowercase such fields? e.g.
geo.continent_code
geo.postal_code
geo.timezone

File: config/enrichments/07_lowercase.conf (hunk 1 above)
Comment (KrishnanandSingh) [reply config/enrichments/07_lowercase.conf:281]: Please look for all such long and float fields

**Review by KrishnanandSingh — CHANGES_REQUESTED** [review]

File: config/enrichments/07_lowercase.conf (hunk 2 above)
Comment (verbecee) [reply config/enrichments/07_lowercase.conf:45]: If it is safe to assume ECS is America-centric, postal codes are always numbers. In other countries that have postal codes, are would the postal code "203foo" point to a different location than "203FOO?"

The examples in ECS for geo.continent_code is "NA" and the timezone is "America/Argentina/Buenos_Aires." Since there isn't a substantive difference in "na" vs "NA" (there isn't another continent called "na"), I can't think of a harmful reason to lowercase them. I think one negative consequence of leaving them uppercase is that other instances of regional data - for example, in host domains - are already lowercased.

File: config/processors/syslog_log_audit_checkpoint.operations.conf [hunk 3]
```diff
@@ -38,7 +38,7 @@ filter {
     }
   }
 
-  if [rest_msg] =~ "illegal pam password attempt" {
+  if [rest_msg] =~ "illegal (pam|PAM) password attempt" {
```
Comment (KrishnanandSingh) [inline config/processors/syslog_log_audit_checkpoint.operations.conf:41]: @Akhila-Y This is unrelated. Please revert this change in this PR. You can create another PR to add this change.

**Review by KrishnanandSingh — CHANGES_REQUESTED** [review]
